readyplayerme/rpm-unity-sdk-avatar-loader
# [3317981848] Add avatar component

## [3317981848](https://ready-player-me.monday.com/boards/2563815861/pulses/3317981848)

## Changes

#### Added

-  Added `AvatarComponent` which contains `AvatarId` and `AvatarMetaData`

## How to Test

- Load an avatar using avatar loader window and observe the `AvatarComponent` attached to the gameObject


## Checklist


## QA Testing

-   Mention any useful test cases for this feature and add to QA Testing documentation.

## Details

-   If more details are necessary to support the description with images and videos add them here.
